modules/networking/default.nix

```diff
@@ -94,6 +95,16 @@ in
       default = [];
       description = lib.mdDoc "The list of search paths used when resolving domain names.";
     };
+
+    networking.wakeOnLan = mkOption {
```

Comment:
Name of option and placement inspired by the NixOS option: [`networking.interfaces.<name>.wakeOnLan.enable`](https://search.nixos.org/options?channel=23.05&show=networking.interfaces.%3Cname%3E.wakeOnLan.enable)

Reply:
Would it make sense to append `.enable`?